pkgs/development/python-modules/playwright/default.nix

```diff
@@ -51,9 +51,7 @@ buildPythonPackage rec {
     git config --global user.name "nixpkgs"
     git commit -m "workaround setuptools-scm"
 
-    substituteInPlace pyproject.toml \
-      --replace-fail 'requires = ["setuptools==75.6.0", "setuptools-scm==8.1.0", "wheel==0.45.1", "auditwheel==6.2.0"]' \
-                     'requires = ["setuptools", "setuptools-scm", "wheel"]'
+    sed -i -e 's/requires = \["setuptools==.*", "setuptools-scm==.*", "wheel==.*", "auditwheel==.*"\]/requires = ["setuptools", "setuptools-scm", "wheel"]/' pyproject.toml
```

Comment:
The versions are updated almost every release so substitute is not suitable here